misc.go

```diff
@@ -208,15 +211,19 @@ func readPairs(r io.Reader) ([][]string, error) {
 				}
 				buf = append(buf, r)
 			}
-			pair[i] = string(buf)
+			arr[i] = string(buf)
 		}
 
-		pairs = append(pairs, pair)
+		pairs = append(pairs, arr)
```

Comment:
I think you might want to rename this variable:

```suggestion
		arrs = append(arrs, arr)
```